config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-apps.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# generates wg-k8s-infra app deployment job configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+readonly OUTPUT="${SCRIPT_DIR}/wg-k8s-infra-apps.yaml"
+# list of subdirs in kubernetes/k8s.io/apps
+readonly APPS=(
+    cert-manager
```

Comment:
I don't think there is a need to include it since we plan to shut it down.

Reply:
removed